### ISSUE TITLE
The game crash when ice enemies appear

### ISSUE THREAD

**Issue by SpardaXV** [elhobbs/hexen#1]
First of all, congratulations on the port, it works very well and is very enjoyable to have map and inventory on the bottom screen. I think it's a great contribution for 3ds.

The bug is quite simple, in one of the first levels there is a time when an ice monster appears (apparently it is called wendingo, attached photo), whenever one of these enemies appears the game is immediately locked with a frozen screen .

I am using a new 3ds, I use the latest 0.7.2 alpha version in the .cia version.

![HXN_Wendigo1](https://user-images.githubusercontent.com/47571088/70855977-cfb68700-1ed3-11ea-8001-fd1414ae62b2.png)

PD: When the game crashes, the following message appears on the lower screen: "sound shutdown complete Z_Malloc: failed on allocation of 196 bytes"

**Comment by elhobbs** [comment]
thanks for reporting the issue. I create a new 0.7.3 alpha release that seems to fix the issue for me. test it out and let me know if the issue is resolved for you as well.

**Comment by SpardaXV** [comment]
It works perfect now, it took me a while to check it because the save files from the previous version don't work (it's the first time I play hexen), but I've reached an ice zone with many of those enemies and it worked fine, I have also made several saves and loaded them, it seems very stable. Great job, congratulations and thanks.

On the other hand, it is a minor detail, but the banner sound saturates the attack because the sound wave is split (that's why a 'click' is heard on the speakers). I have edited the envelope in the attack of the two .wav sounds so that they sound soft, use them as you wish for future updates, I leave them here.



[BannerSounds.zip](https://github.com/elhobbs/hexen/files/3964982/BannerSounds.zip)

**Comment by elhobbs** [comment]
thank you for confirming that this resolved the crash issue for you. I will close this issue.

I rebuilt the cia using the wav files you provided and they still click.

**Comment by SpardaXV** [comment]
@elhobbs Ah you're right, sorry, THNDR2 is also cut in the middle of the sound. I upload it again, now it should be fine.
[THNDR2.zip](https://github.com/elhobbs/hexen/files/3965403/THNDR2.zip)

**Comment by SpardaXV** [comment]
@elhobbs Sorry for not writing before, the game banner still has glitch in the audio.

I apologize for not being able to fix this, I do not understand why the banner continues to click at the beginning of the sound, I applied a fade-in, I checked that the audio wave had no breakage and I tried the .wav on several devices to make sure that the reproduction was good, I don't understand why in 3ds it still fails, it shouldn't.

Maybe the easiest thing is to change the audio track, but when I have some time, I will install arch linux on an old computer and try to compile the game on my own, so I can check the different .wav that I do, I like the current sound of lightning.

Thank you for your work, the lack of time has not allowed me to play and I have not finished the game, but I advanced a lot without any problems, surely the game can be finished (I will check it out).
